core/controllers/base_test.py

```diff
@@ -199,8 +199,9 @@ def test_requests_for_invalid_paths(self):
     def test_redirect_in_logged_out_states(self):
         """Test for a redirect in logged out state on '/'."""
 
-        response = self.get_html_response('/', expected_status_int=302)
-        self.assertIn('splash', response.headers['location'])
+        response = self.get_html_response('/')
```

Comment:
The test name says it should be a redirect url but it's not the same, right?

Reply:
Changed.